.drone.yml

```diff
@@ -71,9 +72,12 @@ steps:
       - make build-contract DOCKER_TAG=$DOCKER_TAG
       - make build-client DOCKER_TAG=$DOCKER_TAG
       - export NODES=3
-      - SGX_MODE=HW docker-compose -f docker-compose.yml -f docker-compose.hw.yml up --scale worker=$NODES --exit-code-from client & sleep 240 && docker-compose exec -T client make test && export RESULT=$? || export RESULT=$?
-      - docker-compose -f docker-compose.yml -f docker-compose.hw.yml down -v --rmi all || true
-      - if [ $RESULT -ne 0 ]; then exit 1; fi
+      - SGX_MODE=HW docker-compose -f docker-compose.yml -f docker-compose.hw.yml up --scale worker=$NODES &
+      - sleep 240
+      - docker-compose exec -T client make test
```

Comment:
@assafmo: if any of the regular tests fail here, nothing is going to catch that error, so the CI will report success when the tests will fail. The `docker-compose exec` does not return the error code from the command it executes.

Reply:
@lacabra Are you sure?
```bash
➜  docker-environment git:(develop) docker-compose exec -T client bash -c 'exit 7' ; echo $?
7
```

Reply:
Ok, I'm convinced. I could have sworn that wasn't the case before, but it clearly works now. Thank you